pages/reference.mdx

```diff
@@ -309,6 +309,308 @@ The ability to view APM traces is only available to Discord employees.
+## Client Properties
+
+Client properties, or "super properties", contain tracking information about the client that is used for analytics and A/B testing purposes. These properties are sent in the Gateway [Identify](/topics/gateway-events#identify) payload and included with every HTTP request using the `X-Super-Properties` header as a base-64 encoded JSON object.
+
+<Alert type="warn">
+
+While including this header is not required, it is highly recommended due to its significance in anti-abuse systems. Additionally, many experimental features require a recent client build number to be specified in this header to function.
+
+</Alert>
+
+###### Client Properties Structure
+
+Due to the nature of client properties, the structure of this object is not well-defined, and no field is truly required.
+Additionally, types of fields are not verified, and all documented enums are merely conventions. Fields are marked as required if it's observed that they are sent in all official client properties.
+
+| Field                                           | Type     | Description                                                                                                                                                                            |
+| ----------------------------------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| os                                              | string   | The [operating system of the client](#operating-system-type)                                                                                                                           |
+| os_version                                      | string   | The operating system version (kernel version for Linux, SDK version for Android)                                                                                                       |
+| os_sdk_version? **(deprecated)**                | string   | The Android operating system SDK version                                                                                                                                               |
+| os_arch?                                        | string   | The architecture of the operating system                                                                                                                                               |
+| app_arch?                                       | string   | The architecture of the desktop application                                                                                                                                            |
+| browser ^1^                                     | string   | The [browser the client is using](#browser-type)                                                                                                                                       |
```

Comment:
Only `os` and `client_build_number` are needed to unlock experiments with a minimum client version requirement. `browser` doesn't seem to make a difference.

```suggestion
| os ^1^                                             | string   | The [operating system of the client](#operating-system-type)                                                                                                                           |
| os_version                                      | string   | The operating system version (kernel version for Linux, SDK version for Android)                                                                                                       |
| os_sdk_version? **(deprecated)**                | string   | The Android operating system SDK version                                                                                                                                               |
| os_arch?                                        | string   | The architecture of the operating system                                                                                                                                               |
| app_arch?                                       | string   | The architecture of the desktop application                                                                                                                                            |
| browser                                      | string   | The [browser the client is using](#browser-type)                                                                                                                                       |
```